### ISSUE TITLE
How to select best checkpoint when fine-tuning on GSM8k 

### ISSUE THREAD

**Issue by dineshkh** [sail-sg/sdft#14]
How do you select best checkpoint when fine-tuning on GSM8k ?

**Comment by rickyang1114** [comment]
Thank you for your interest. We use the final checkpoint after fine-tuning.

**Comment by dineshkh** [comment]
Thanks for your reply. So there is no validation set for selecting the best checkpoint?

**Comment by rickyang1114** [comment]
Yes.
